ansible/module_utils/port_utils.py

```diff
@@ -198,6 +198,11 @@ def get_port_alias_to_name_map(hwsku, asic_id=None):
         else:
             for i in range(1,9):
                 port_alias_to_name_map["Ethernet1/%d" % i] = "Ethernet%d" % ((i - 1) * 4)
+    elif hwsku == "B6510-48VS8CQ":
+        for i in range(1,48):
```

Comment:
range(1,49) to include 48 as well

Reply:
Our device Ethernet1~Ethernet48 are 25G, Ethernet49~Ethernet56 are 100G

Reply:
range(1,48), not range(1,49)

Reply:
Probably I didn't specify my comment clearly. port 48 also need to get included in your first range. if you specify range(1,48), only ports 1 to 47 are included. python range does not include the end value

Reply:
Oh! My mistake, I got it.

Reply:
Retesting vs now